static/dependencies.config.php

```diff
@@ -75,13 +75,13 @@
 	Util\ConfigFileLoader::class => [
 		'shared'          => true,
 		'constructParams' => [
-			[Dice::INSTANCE => '$basepath'],
+			[Dice::INSTANCE => '$basepath']
```

Comment:
Please keep the trailing comma, it improves the diff readability when adding an item at the end of arrays.